Makefile

```diff
@@ -13,7 +15,7 @@ DOCKERFILES = $(sort \
 build: $(patsubst %, build-%, $(PLUGINS))
 
 build-%:
-	cd cmd/$(subst -,/,$*) && go build
```

Comment:
why did you change it to install?

Reply:
today the project is small, so rebuilding from scratch is not a big deal. but if you have large codebase, "go install" make sure that you only compile modified packages

Reply:
did not know that, thank you